developer/src/tike/actions/dmActionsDebugger.pas

```diff
@@ -0,0 +1,428 @@
+  Dependencies:     
+
+  Bugs:             
+  Todo:             
+  Notes:            
+  History:          23 Aug 2006 - mcdurdin - Initial version
+                    14 Sep 2006 - mcdurdin - Add debugger actions
+                    28 Sep 2006 - mcdurdin - Added Test Keyboard menu item
+                    06 Oct 2006 - mcdurdin - Add Test Keyman Web keyboard
+                    04 Dec 2006 - mcdurdin - Implement all debug actions
+                    04 Dec 2006 - mcdurdin - Localize
+                    04 Dec 2006 - mcdurdin - Add keyboard upload
+                    22 Jan 2007 - mcdurdin - Reset debugger before compiling
+                    23 Aug 2007 - mcdurdin - I962 - Fix crash when uploading a keyboard and pressing Cancel
+                    12 Oct 2007 - mcdurdin - I654 - handle error when installing or uninstalling an invalid keyboard file
+                    16 Jan 2009 - mcdurdin - Add keyboard font helper command (incomplete)
+                    26 Jul 2010 - mcdurdin - I2468 - KeymanWeb Pack elimination, part of standard build
+                    17 Dec 2010 - mcdurdin - I2595 - Remove GnuGetText
+                    18 May 2012 - mcdurdin - I3306 - V9.0 - Remove TntControls + Win9x support
+                    18 May 2012 - mcdurdin - I3323 - V9.0 - Change from Plus-MemoU to Plus-Memo
+                    04 Dec 2013 - mcdurdin - I3655 - V9.0 - Keyboard debugger does not appear to function in 9.0.419.0
+                    21 Feb 2014 - mcdurdin - I4057 - V9.0 - Keyman Developer Keyboard Font dialog helpful to reduce font confusion
+                    04 Nov 2014 - mcdurdin - I4504 - V9.0 - Consolidate the compile action into single command
+                    04 May 2015 - mcdurdin - I4686 - V9.0 - Refactor compile into project file action
+                    04 May 2015 - mcdurdin - I4687 - V9.0 - Split project UI actions into separate classes
+                    
+*)
+unit dmActionsDebugger;  // I3306  // I3323
+
+interface
+
+uses
+  SysUtils,
+  Classes,
+  ActnList,
+  UfrmSelectSystemKeyboard,
+  ImgList,
+  Controls,
+  Dialogs,
+  System.Actions;
+
+type
+  TmodActionsDebugger = class(TDataModule)
+    actionsDebug: TActionList;
+    actDebugStartDebugger: TAction;
+    actDebugSetClearBreakpoint: TAction;
+    actDebugStopDebugger: TAction;
+    actDebugSingleStepMode: TAction;
+    actDebugStepForward: TAction;
+    actDebugRun: TAction;
+    actDebugSelectSystemKeyboard: TAction;
+    actDebugANSITestMode: TAction;
+    actDebugViewElements: TAction;
+    actDebugViewCallStack: TAction;
+    actDebugViewDeadkeys: TAction;
+    actDebugViewRegressionTesting: TAction;
+    actDebugTestMode: TAction;
+    actDebugDebuggerMode: TAction;
+    actDebugPause: TAction;
+    actDebugViewFont: TAction;
+    actDebugViewDefaultFont: TAction;
+    dlgFont: TFontDialog;
+    actDebugViewState: TAction;
+    actDebugSwitchToDebuggerWindow: TAction;
+    procedure actionsKeyboardEditorUpdate(Action: TBasicAction;
```

Comment:
As best as I can tell, the method above is new, but all that follow in this list have been migrated from `dmActionsKeyboardEditor` within this commit.